email-templates/package.json

```diff
@@ -16,7 +16,8 @@
   "license": "MIT",
   "dependencies": {
     "azure-storage": "^2.8.2",
-    "foundation-emails": "^2.2.1"
+    "foundation-emails": "^2.2.1",
+    "npm": "^6.3.0"
```

Comment:
I don't think I can go for this -- just remove it and we'll try to fix another way.